joss/paper.md

```diff
@@ -118,7 +120,7 @@ parallel if supported by hardware). The fit results across times and
 channels are available on demand with the same syntax used to access
 results in a single fit object and the results are returned as tidy
 indexed `pandas.DataFrames` for further analysis, visualization, and
```

Comment:
would be nice to cite pandas (the paper), and also numpy and other **crucial** packages that you depend on.

Reply:
Done, the References now include pandas, numpy, matplotlib, statsmodels, pymer4, rpy2, R, lme4, lmerTest.